src/test/java/io/tus/java/client/TestTusUploader.java

```diff
@@ -69,6 +68,9 @@ public void testTusUploader() throws IOException, ProtocolException {
         assertEquals(-1, uploader.uploadChunk(5));
         assertEquals(11, uploader.getOffset());
         uploader.finish();
+        String log = mockServer.retrieveLogMessages(new HttpRequest());
+        System.out.println("x");
+
```

Comment:
These debugging statement can be removed, can't they?

Reply:
... yes